reference/functions/findfiles_up.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: findfiles_up
+published: true
+tags: [reference, file functions, functions, findfiles_up, files, glob]
+---
+
+[%CFEngine_function_prototype(path, glob, level)%]
+
+**Return type:** `data array`
+
+**Arguments:**
```

Comment:
Hey hey, we have a macro for this.

```
[%CFEngine_function_attributes(regex, string, backref)%]
```

As seen rendered https://docs.cfengine.com/docs/master/reference-functions-regextract.html and the source here https://github.com/cfengine/documentation/blob/master/reference/functions/regextract.markdown

Reply:
Oh, I updated functions/template.markdown to include that.